packages/forms/src/elements/common/Field.tsx

```diff
@@ -36,9 +37,17 @@ export const Field = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElem
         setIsLabelActive,
         isLabelHovered,
         setIsLabelHovered,
-        multiThumbRangeRef
+        multiThumbRangeRef,
+        getInputProps: (...args: any[]) => {
+          const { 'aria-describedby': describedBy, ...inputProps } = getInputProps(...args);
+
+          if (hasHint) inputProps['aria-describedby'] = describedBy;
```

Comment:
Would `inputProps['aria-describedby']` deliberately need to be set to `undefined` if `hasHint` is `false` on state change?

Reply:
No it shouldn't need to be explicitly set because the `getInputProps` function gets called on every render. We are overriding the useField hook outputs (`'aria-describedby'`) that to check if the component exists in this instance.